service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterStatementMapper.java

```diff
@@ -0,0 +1,56 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import lombok.RequiredArgsConstructor;
+import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
+import uk.nhs.adaptors.gp2gp.ehr.utils.DateFormatUtil;
+import uk.nhs.adaptors.gp2gp.ehr.utils.TemplateUtils;
+
+import org.hl7.fhir.dstu3.model.Encounter;
+import org.hl7.fhir.dstu3.model.Period;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.github.mustachejava.Mustache;
+
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Component
+public class EncounterStatementMapper {
+    private static final Mustache ENCOUNTER_STATEMENT_TO_EHR_COMPOSTION_TEMPLATE =
+        TemplateUtils.loadTemplate("ehr_encounter_statement_to_ehr_composition_template.mustache");
+    private static final String COMPLETE_CODE = "COMPLETE";
+    private static final String DEFAULT_TIME_VALUE = "<center nullFlavour=\"UNK\"/>";
+    private static final String EFFECTIVE_LOW_TEMPLATE = "<low value=\"%s\"/>";
+    private static final String EFFECTIVE_HIGH_TEMPLATE = "<high value=\"%s\"/>";
+
+    private final RandomIdGeneratorService randomIdGeneratorService;
+
+    public String mapEncounterToEncounterStatement(Encounter encounter) {
+        var encounterStatementTemplateParameters = EncounterStatementTemplateParameters.builder()
+            .encounterStatementId(randomIdGeneratorService.createNewId())
+            .status(COMPLETE_CODE);
```

Comment:
From the mapping spreadsheet the status code is fixed to be "COMPLETE" so would it ok to be hardcoded in the template instead of building this? 
Currently this is hardcoded in the templates of the other mappers, thus may need to refactor those depending on the decision.

Reply:
Both solutions are correct. I put it here if we needed in the future to set a different status.